deploy/31_deploy_tokenholder_governor.ts

```diff
@@ -23,6 +23,18 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
     log: true,
   })
 
+  if (hre.network.tags.etherscan) {
+    await hre.ethers.provider.waitForTransaction(
+      timelock.transactionHash,
+      5,
+      300000
```

Comment:
I understand the occasional need for this, but there's nothing inherent to `TokenholderGovernor` that requires an extra waiting time. It can happen to other deployments with some small probability. Can we instead add this to all Etherscan verification calls? Ideally, we can introduce it as part of a helper function.